openquake/hazardlib/gsim_lt.py

```diff
@@ -272,8 +275,11 @@ def __fromh5__(self, array, dic):
                 gsim = valid.gsim(branch['uncertainty'], dirname)
                 for k, v in gsim.kwargs.items():
                     if k.endswith(('_file', '_table')):
-                        arr = numpy.asarray(dic[os.path.basename(v)][()])
-                        gsim.kwargs[k] = io.BytesIO(bytes(arr))
+                        if v == None:
+                            pass
```

Comment:
Why this is happening now? What is the test causing the `_file` to be None?

Reply:
I have removed this conditional statement. This was added for testing the non-ergodic gsim without the volcanic polygon geoJSON to check the predicted ground-motions were not equal to those computed if the volcanic attenuation is considered (now the comparison is simpler in the unit test we use the regular sslab version and the non-ergodic sslab version, so this line of code is no longer needed and has been removed)